src/utils/debug/texture-selector-utils.ts

```diff
@@ -22,7 +22,31 @@ export function selectOriginalTextureForTileset() {
   }
 }
 
-export function selectDebugTextureForTile(tile, uvDebugTexture) {
+const cropTexture = async (texture, width, height) => {
```

Comment:
Add TSDoc comments and types for the function

Reply:
Done